tests/system/lab/test

```diff
@@ -186,12 +188,16 @@ setup_north_pi_egress () {
     fi
 
     echo
-
+    
+    # Wait for OMF to start  up properly
+    sleep 60
```

Comment:
Is this enough for all platforms? We should avoid hardcoded sleeps and way to check with better way.